File: HotDesk_SPICE/Backup/Library/Library.Control.Base/Page.vb
Public MustInherit Class Page
    Inherits System.Web.UI.Page

    Private _controlPanel As System.Web.UI.WebControls.PlaceHolder = Nothing
    Public _list As New Collections.Generic.List(Of String)

    Public Property ControlPanel() As System.Web.UI.WebControls.PlaceHolder
        Get
            Return _controlPanel
        End Get
        Set(ByVal value As System.Web.UI.WebControls.PlaceHolder)
            _controlPanel = value
        End Set
    End Property

    Public Sub Remove(ByVal Ctrl As System.Web.UI.Control)
        Ctrl.Visible = False
  
    End Sub
    Private _Action As Int16
    Public Property Action() As Int16
        Get
            Return _Action
        End Get
        Set(ByVal value As Int16)
            _Action = value
        End Set
    End Property
    Private _CurrentStep As Int16
    Public Property CurrentStep() As Int16
        Get
            Return _CurrentStep
        End Get
        Set(ByVal value As Int16)
            _CurrentStep = value
        End Set
    End Property

    Private _Worksno As String
    Public Property Worksno() As String
        Get
            Return _Worksno
        End Get
        Set(ByVal value As String)
            _Worksno = value
        End Set
    End Property

    Private _CelNo As String
    Public Property CelNo() As String
        Get
            Return _CelNo
        End Get
        Set(ByVal value As String)
            _CelNo = value
        End Set
    End Property

    Private _CompCode As String
    Public Property CompCode() As String
        Get
            Return _CompCode
        End Get
        Set(ByVal value As String)
            _CompCode = value
        End Set
    End Property

    Private _Reqno As String
    Public Property Reqno() As String
        Get
            Return _Reqno
        End Get
        Set(ByVal value As String)
            _Reqno = value
        End Set
    End Property

End Class

File: TMS_HotDesk/Library/Library.Control.Base/Page.cs
using System.Collections.Generic;

public abstract class Page : System.Web.UI.Page
{
    private System.Web.UI.WebControls.PlaceHolder _controlPanel = null;
    public List<string> _list = new List<string>();

    public System.Web.UI.WebControls.PlaceHolder ControlPanel
    {
        get { return _controlPanel; }
        set { _controlPanel = value; }
    }

    public void Remove(System.Web.UI.Control ctrl)
    {
        ctrl.Visible = false;
    }

    private short _Action;
    public short Action
    {
        get { return _Action; }
        set { _Action = value; }
    }

    private short _CurrentStep;
    public short CurrentStep
    {
        get { return _CurrentStep; }
        set { _CurrentStep = value; }
    }

    private string _Worksno;
    public string Worksno
    {
        get { return _Worksno; }
        set { _Worksno = value; }
    }

    private string _CelNo;
    public string CelNo
    {
        get { return _CelNo; }
        set { _CelNo = value; }
    }

    private string _CompCode;
    public string CompCode
    {
        get { return _CompCode; }
        set { _CompCode = value; }
    }

    private string _Reqno;
    public string Reqno
    {
        get { return _Reqno; }
        set { _Reqno = value; }
    }
}
